Classes/Queue/RabbitQueue.php

```diff
@@ -258,7 +277,7 @@ protected function queue(string $payload, array $options = [], int $numberOfRele
 
         $headers = new AMQPTable($headerOptions);
         $message->set('application_headers', $headers);
-        $this->channel->basic_publish($message, $this->exchangeName, $this->routingKey !== '' ? $this->routingKey : $this->name);
+        $this->channel->basic_publish($message, $this->exchangeName, $this->routingKey);
```

Comment:
This change might be breaking.

I think it should be possible to publish messages without a routing key. Maybe by adding a new configuration option for that?

Reply:
Yeah, let's just release a major then?